dipy/utils/multiproc.py

```diff
@@ -40,3 +41,40 @@ def determine_num_processes(num_processes):
         return 1
 
     return num_processes
+
+
+def disable_np_threads():
+    """Reduce OPENBLAS and MKL thread numbers.
```

Comment:
Do you want to make the number of threads an argument to this function? Right now, this always reduces to 1, but maybe sometimes we want to reduce down to number of cpus?